### PR TITLE
Implementation of the simple business card

### PR DESCRIPTION
**Pull request by ezequielsetzesglobant**
A simple business card application was implemented.

Image:
Screen Shot 2022-04-11 at 12 14 36

Test:
Screen Shot 2022-04-11 at 12 16 01

### CODE REVIEW

**Review by FranciscoGaraventa — APPROVED** [review]
LGTM!

**Review by MarcosCGlobant — APPROVED** [review]
LGTM

**Review by Juliana-Dimeglio — APPROVED** [review]

File: lib/home_page.dart [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+class HomePage extends StatelessWidget {
+  const HomePage({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.amber,
+      body: SafeArea(
+          child: Center(
+        child: Column(
```
Comment (Juliana-Dimeglio) [inline lib/home_page.dart]: Please review indentation. You can use option + command + L to indent all the code of the file correctly

**Review by Juliana-Dimeglio — CHANGES_REQUESTED** [review]

File: lib/home_page.dart (hunk 1 above)
Comment (ezequielsetzesglobant) [reply lib/home_page.dart]: done

**Review by Juliana-Dimeglio — APPROVED** [review]
LGTM!

**Pull request merged**